tools/docker/tests/test_flows.sh

```diff
@@ -20,6 +20,17 @@ error=0
 
 bridge_name=br-lan
 
+get_host_bridge_name() {
+    docker network inspect prplMesh-net --format="br-{{ printf \"%.12s\" .Id }}"
+}
+
+kill_tcpdump() {
+    if [ -n "$TCPDUMP_PROC" ]; then
+        status "Terminating tcpdump"
+        kill $TCPDUMP_PROC;
```

Comment:
Redundant semicolon (are you a C programmer? :-)

TCPDUMP_PROC should also be unset or set to empty after the kill, so we don't kill it twice (and the second time it may already be a new process...)